site/specs/numbers.json

```diff
@@ -15546,125 +15717,113 @@
+            "description": "Checks the order's creation date against this value. Orders that have a creation date after this date will be included. Format is yyyy-MM-dd.",
             "in": "query",
             "required": false,
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "2013-10-22"
           },
           {
             "name": "createdDateTo",
-            "description": "<p>Checks the order's creation date against this value. Orders that have a creation date before this date will be included. Format is yyyy-MM-dd</p>\n",
+            "description": "Checks the order's creation date against this value. Orders that have a creation date before this date will be included. Format is yyyy-MM-dd.",
             "in": "query",
             "required": false,
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "2013-10-25"
           },
           {
             "name": "lastModifiedAfter",
-            "description": "<p>Checks the order's last modified date against this value. Orders that have a modification date after this date will be included. Format is yyyy-MM-dd</p>\n",
+            "description": "Checks the order's last modified date against this value. Orders that have a modification date after this date will be included. Format is yyyy-MM-dd.",
             "in": "query",
             "required": false,
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "2013-10-25"
           },
           {
             "name": "lastModifiedBy",
-            "description": "<p>This is the user-name of the user that last modified the TN Option Order.</p>\n",
+            "description": "This is the user-name of the user that last modified the TN Option Order.",
             "in": "query",
             "required": false,
             "schema": {
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "smckinnon"
           },
           {
             "name": "orderIdFragment",
-            "description": "<p><b class=\"required\">*</b> This search parameter represents the internal Bandwidth Dashboard API ID that has been assigned to the TN Option Order.  This parameter is the first few characters of the internal ID - the entire ID does not need to be specified.</p>\n",
+            "description": "<b>*</b> This search parameter represents the internal Bandwidth Dashboard API ID that has been assigned to the TN Option Order. This parameter is the first few characters of the internal ID - the entire ID does not need to be specified.",
```

Comment:
let's remove the asterisk and marked "required": true if it's required

Reply:
It's not a required field. It's a specific rule for that query.  ```If no date range or specific query parameter (marked by <b>*</b> below) is provided, the order results will be limited to the last two years.```

Reply:
ahh